src/main/java/blackjack/view/OutputView.java

```diff
@@ -0,0 +1,100 @@
+
+    public void printDealCards(final ParticipantResponse dealer, final List<ParticipantResponse> players,
+                               final int count) {
+        System.out.println();
+        System.out.println(format("{0}와 {1}에게 {2}장을 나누었습니다.",
+                dealer.getName(), getPlayerNamesFormat(players), count));
+
+        printHandedCardsWithoutScore(dealer);
+        players.forEach(this::printHandedCardsWithoutScore);
+    }
+
+    private String getPlayerNamesFormat(final List<ParticipantResponse> players) {
+        return players.stream()
+                .map(ParticipantResponse::getName)
+                .collect(Collectors.joining(", "));
+    }
+
+    public void printHandedCardsWithoutScore(final ParticipantResponse participant) {
+        System.out.println(getHandedCards(participant));
+    }
+
+    private String getHandedCards(final ParticipantResponse participant) {
+        final CardsResponse cardsResponse = participant.getCardsResponse();
+        final List<String> cardInfos = cardsResponse.getCardInfos();
+
+        return format("{0}카드: {1}", participant.getName(), getCardInfosFormat(cardInfos));
+    }
+
+    private String getCardInfosFormat(final List<String> cardInfos) {
+        return String.join(", ", cardInfos);
+    }
+
+    public void printDealerCardDrawn(final DealerStateResponse dealerStateResponse) {
+        if (dealerStateResponse.isDrawable()) {
+            System.out.println();
+            System.out.println(format("딜러는 {0}이하라 한장의 카드를 더 받았습니다.", dealerStateResponse.getLimit()));
+            System.out.println();
+        }
+    }
+
+    public void printHandedCardsWithScore(final ParticipantResponse participant) {
+        final CardsResponse cardsResponse = participant.getCardsResponse();
+        final int totalScore = cardsResponse.getTotalScore();
+
+        System.out.println(format("{0} - {1}", getHandedCards(participant), getScoreFormat(totalScore)));
+    }
+
+    private String getScoreFormat(final int totalScore) {
+        return format("결과: {0}", totalScore);
+    }
+
+    public void printResult(final String dealerName, final List<PlayerResultResponse> players) {
+        System.out.println();
+        System.out.println("## 최종 승패");
+        System.out.println(format(RESULT_FORMAT, dealerName, getDealerResult(players)));
+        players.forEach(this::printPlayerResult);
+    }
+
+    private String getDealerResult(final List<PlayerResultResponse> players) {
+        final Map<Result, Integer> playerResult = initPlayerResult();
+        for (PlayerResultResponse player : players) {
+            final Result result = player.getResult();
+            playerResult.put(result, playerResult.getOrDefault(result, 0) + 1);
+        }
+
+        return format("{0}승 {1}무 {2}패", playerResult.get(LOSE), playerResult.get(DRAW), playerResult.get(WIN));
+    }
+
+    private Map<Result, Integer> initPlayerResult() {
+        final Map<Result, Integer> playerResult = new HashMap<>();
```

Comment:
```suggestion
        final Map<Result, Integer> playerResult = new EnumMap<>();
```

HashMap보다 EnumMap이 더 적절할 것 같아요.
두 자료구조는 어떤 차이가 있을까요?

Reply:
오.. 찾아보다가 Key의 값이 Enum인 경우 HashMap보다는 EnumMap을 사용하라는 글을 봤습니다. Enum이라는 특수한 조건때문에 일반적인 Map보다 기능 시간복잡도가 더 빠르다는 이점이 있다는 것 같네요. 이번에는 EnumMap으로 바꿔보았습니다!